rocky/reports/report_types/aggregate_organisation_report/recommendations.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+
+<section id="recommendations">
+    <div>
+        <h2>{% translate "Recommendations" %}</h2>
+        <p>
+            Er zijn <i>n</i> kwetsbaarheden gevonden op <i>m</i> systemen. <i>O</i> systemen missen belangrijke basisbeveiligingsmaatregelen.
+        </p>
+        <ul>
+            <li>Update the web server of system IP (domain name)</li>
+            <li>Install the firewall in the OS of system IP (domain name)</li>
+            <li>
+                Check if <a href="">SPF [1]</a> is properly installed on all mail servers
+            </li>
```

Comment:
This seems way to specific, and not based on the actual data?

Reply:
Yes, this is also an example. Filling this in with real recommendations